wordpress-mobile/WordPress-Android
# Issue/13268 persist activity type filters in parent vm

Partially fixes #13268 

Merge instructions:
1. Make sure https://github.com/wordpress-mobile/WordPress-Android/pull/13483 is merged
2. Remove not ready for merge label
3. Merge this PR

This PR adds support for persisting checked items when the user sets a filter on Activity Type filter screen and opens it again from Activity log. The items are intentionally not persisted when the user leaves the Activity Log feature completely.

To test:
1. Open My Site
2. Open Activity Log
3. Click on "Activity Type" chip
4. Select some items
5. Click on "x" in the top left corner
6. Click on "Activity Type" chip again
7. Notice none of the items is checked
8. Select some items
9. Click on Apply
10. Click on "Activity Type" chip again
11. Notice previously selected items are checked
12. Click on apply
13. Leave the Activity Log
14. Open Activity Log
15. Click on "Activity Type" chip again
16. Notice none of the items is checked

Test anything else you can think of (configuration change), clear action, leaving the screen, etc..

PR submission checklist:
